File: TestProject/TestProject/ViewController.m
//
//  ViewController.m
//  TestProject
//
//  Created by Rogelio Gudino on 2/3/16.
//  Copyright © 2016 Rogelio Gudino. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = NSLocalizedStringWithDefaultValue(@"main-title", nil, [NSBundle mainBundle], @"Main", @"Main’s title.");
}

@end

File: LocalizationsList/Localization/ViewController.swift
//
//  ViewController.swift
//  Localization
//
//  Created by Rogelio Gudino on 9/9/15.
//  Copyright (c) 2015 Rogelio Gudino. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var label: UILabel?
    override func viewDidLoad() {
        super.viewDidLoad()
        let t = "Test"
        label?.text = NSLocalizedString("\(t)", comment: "")
    }
}

